airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/task/DestinationTaskLauncherTest.kt

```diff
@@ -257,6 +279,12 @@ class DestinationTaskLauncherTest {
         launch { taskRunner.run() }
         taskLauncher.handleSetupComplete()
         mockOpenStreamTaskFactory.streamHasRun.values.forEach { it.receive() }
+
+        repeat(mockOpenStreamTaskFactory.streamHasRun.size) {
```

Comment:
nit: this probably should assert on an exact count of exceptions? something like
```kotlin
assertAll(
  { assertEquals(2, mockOpenStreamTaskFactory.streamHasRun.size) },
  { ... the thing you already have here }
)
```